src/main/java/crawlercommons/sitemaps/extension/VideoAttributes.java

```diff
@@ -519,6 +525,159 @@ public void setPrice(Float price) {
 
     }
 
+    /**
+     * <blockquote>If you publish your video as a series of raw videos (for
```

Comment:
Maybe we should add the missing part from the XSD here: `Use <video:content_segment_loc> only in conjunction with <video:player_loc>. `. Otherwise, it reads a bit weird :)